src/main/java/com/github/piedpiper/node/aws/AWSStepFunctionsFactory.java

```diff
@@ -0,0 +1,19 @@
+package com.github.piedpiper.node.aws;
+
```

Comment:
Why inject factory ? why can't you inject the AWSStepFunctionsClient itself as a @Provider